cmd/edit/addon/cmd.go

```diff
@@ -46,7 +46,20 @@ var Cmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		if len(argv) > 0 && (strings.HasPrefix(argv[0], "--") || strings.HasPrefix(argv[0], "-")) {
+		
+		idMissing := true
+		if len(argv) > 0 {
+			for i, arg := range argv {
+				if i == 0 && !(strings.HasPrefix(arg, "--") || strings.HasPrefix(arg, "-")) {
+					idMissing = false
+				}
+				if !strings.HasPrefix(arg, "--") && !strings.HasPrefix(arg, "-") &&
```

Comment:
I think you could just loop through and find an argument that doesn't start with '-'

Reply:
There is an issue with this solution, for example `-c hkepley-cluster` will return true in this case as the args are separated by spaces not `--flag arg`

Reply:
where as it should be false unless `-c hkepley-cluster test-id`